src/pages/blog/components/post-card/PostCard.styled.ts

```diff
@@ -0,0 +1,58 @@
+import Box from "@mui/material/Box";
+import IconButton from "@mui/material/IconButton";
+import { styled } from "@mui/material/styles";
```

Comment:
Same here, always use `@mui/material` for importing styled

Reply:
Fixed